src/mngt/ocf_mngt_flush.c

```diff
@@ -141,56 +141,61 @@ static int _ocf_mngt_get_sectors(ocf_cache_t cache, ocf_core_id_t core_id,
 	uint64_t core_line;
 	ocf_core_id_t i_core_id;
 	struct flush_data *p;
-	uint32_t i, j, dirty = 0;
+	uint32_t i, dirty_found = 0, dirty_total = 0;
```

Comment:
`i` is referring do cache_line_id? Maybe it would be also worth renaming? The same for `p` - if you have any better ideas to name it.